src/System.Windows.Forms/src/System/Windows/Forms/Control.ActiveXImpl.cs

```diff
@@ -43,17 +41,17 @@ private unsafe class ActiveXImpl : MarshalByRefObject, IWindowTarget
             private static readonly int s_adjustingRect = BitVector32.CreateMask(s_uiDead);
 
             private static Point s_logPixels = Point.Empty;
-            private static OLEVERB[] s_axVerbs;
+            private static OLEVERB[]? s_axVerbs;
 
             private readonly Control _control;
             private readonly IWindowTarget _controlWindowTarget;
             private Rectangle? _lastClipRect;
 
-            private IOleClientSite _clientSite;
-            private IOleInPlaceUIWindow _inPlaceUiWindow;
-            private IOleInPlaceFrame _inPlaceFrame;
+            private IOleClientSite? _clientSite;
+            private IOleInPlaceUIWindow? _inPlaceUiWindow;
+            private IOleInPlaceFrame? _inPlaceFrame;
             private readonly ArrayList _adviseList;
```

Comment:
We can probably change this to
```suggestion
            private readonly List<IAdviseSink> _adviseList;
```
...as this will remove associated bang operators . We can do it later though.